### PR TITLE
Localize validation messages, Fixes #36

### LINKED ISSUE #36: Error messages for registering an account are not localized.
Error messages for registering an account are not localized.

Example:
Ange bara en email som inte finns i vårt system, tack (email): ___________
**The email given is not a valid email address.**

**Please enter an email.**

### PR DESCRIPTION
**Pull request by Cactu5**

### CODE REVIEW

**Comment by william-stacken** [comment]
Maybe a test about this could be added to the view tests?

**Comment by Cactu5** [comment]
> Maybe a test about this could be added to the view tests?

Yes, I'll do that.

**Comment by Cactu5** [comment]
I will add the tests after #24 has been merged into dev.

**Comment by Cactu5** [comment]
Tests fail because of #43

File: src/main/resources/i18n/ValidationMessages.properties [hunk 1]
```diff
@@ -1,65 +1,34 @@
...
-
-availability.jobApplication.missing=The job application associated to the availability is missing.
-
-competence.name.length=The name must be between 2 and 30 characters long.
-
-competenceProfile.yearsOfExperience.lessThanZero=The years of experience can not be less than zero.
-competenceProfile.competence.missing=The competence associated to the competence profile is missing.
-competenceProfile.jobApplication.missing=The job application associated to the competence profile is missing.
-
-jobApplication.applicant.missing=The applicant associated to the job application is missing.
```
Comment (william-stacken) [inline src/main/resources/i18n/ValidationMessages.properties]: Out of curiosity, why were all of these messages removed?

Comment (Cactu5) [reply src/main/resources/i18n/ValidationMessages.properties]: The messages are all in English. To follow what we have done previously. All English messages will be placed in a file with the ending `_en` and Swedish translations will be in the default file.

Comment (Cactu5) [reply src/main/resources/i18n/ValidationMessages.properties]: Seems like I accidentally deleted those. They have been added in d7289da

**Review by william-stacken — APPROVED** [review]
Very good 10/10 no ✈️ 🔥

**Pull request merged**